src/main/java/net/trollyloki/murdermystery/game/Game.java

```diff
@@ -542,12 +547,26 @@ public void run() {
             else
                 graceMessage = String.format(plugin.getConfigString("time.grace_warning"), this.graceTime);
         }
+        
+        String potatoMessage = null;
+        if (isRunning() && hotPotatoMode && this.potatoTime >= 0) {
+        	if (this.potatoTime == 0) 
+        		potatoMessage = String.format(plugin.getConfigString("time.potato_burned"), Bukkit.getPlayer(potatoVictim).getName());
+        	// Else if the potato time is <= to the countdown time - countdown time is so it doesn't spam chat too much
+        	else if (this.potatoTime <= plugin.getConfig().getInt("time.potato_countdown"))	
+        		potatoMessage = String.format(plugin.getConfigString("time.potato_warning"), this.potatoTime);
+        }
         // Almost forgot to check if potatomode was on! If I hadn't caught that we'd be killing a null object!
         // Added setting potatoVictim to null if the player died in the kill method
         if (isRunning() && hotPotatoMode && this.potatoTime == 0) {
             if (this.potatoVictim != null) {
                 Player victim = Bukkit.getPlayer(potatoVictim);
                 if (victim != null)
+                	// bam, fireworks
+                	firework = (Firework) victim.getWorld().spawnEntity(victim.getLocation(), EntityType.FIREWORK);
```

Comment:
You can do this without casting if you use `victim.getWorld().spawn(victim.getLocation(), Firework.class)`